protocol-dashboard/src/components/TotalStakedChart/TotalStakedChart.tsx

```diff
@@ -20,14 +25,21 @@ const TotalStakedChart: React.FC<TotalStakedChartProps> = () => {
     labels = totalStaked?.map(s => s.timestamp) ?? null
     data = totalStaked?.map(s => s.count) ?? null
   }
+
+  const { status: trailingTotalStatus, total } = usedLiveTotalStakedAudio()
+  if (total && trailingTotalStatus === Status.Success) {
+    topNumber = AudiusClient.displayShortAud(total)
+  }
+
   return (
     <LineChart
-      title="Total Staked"
+      topNumber={topNumber}
+      title="Global Staked $AUDIO"
       error={error}
       data={data}
       labels={labels}
       selection={bucket}
-      options={[Bucket.ALL_TIME, Bucket.MONTH, Bucket.WEEK]}
+      options={[Bucket.ALL_TIME, Bucket.YEAR, Bucket.MONTH, Bucket.WEEK]}
       onSelectOption={(option: string) => setBucket(option as Bucket)}
```

Comment:
we have special bucket types for the other charts so just checking if we want to same here?